src/app/page.tsx

```diff
@@ -1,7 +1,10 @@
 import { Button } from '@/common/ui/Button/Button'
+import { getUserOrServerRedirect } from '@/common/utils/auth/getUserOrServerRedirect'
 import Link from 'next/link'
 
-export default function Home() {
+export default async function Home() {
+  const user = await getUserOrServerRedirect()
```

Comment:
Can remove the `const user = ` since we don't use it here. Or keep it, if we plan to use it.